src/pages/HomePage.tsx

```diff
@@ -30,10 +30,11 @@ const useStyles = makeStyles({
   },
 });
 
-const HomePage: FC<IProps> = () => {
+const HomePage: FC<IProps> = ({ component: Component }) => {
```

Comment:
@danielsh28 I changed the logic. homePage is now a wrapper component that returns the widget component.

before, homePage had a nested router that redirect to the same url's that "App" was redirecting. 
it didn't work when switching to react-router-dom6. 
i think because nested routes work relatively now.